### PR TITLE
Works by evagrius

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: new/LIT6026NagaraBaqweet.xml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6026NagaraBaqweet" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ነገረ፡ በቍኤት፡ ዘቅዱስ፡ አባ፡ ወግሪስ፡</title>
```
Comment (eu-genia) [inline new/LIT6026NagaraBaqweet.xml:8]: Take care we already have
LIT2022Monast
LIT2121OntheE = CPG 2451
and
LIT2318Senten = CPG 2435

(I assume the ID you created is actually a doublet to LIT2318Senten)


(see EAE II 457b-458ab) 

********
"
The treatise "On the Eight Thoughts (or: Spirits,
Greek logismoiv) of Evilness" (CPG II, no. 2451) –
..... BritLib Orient. 754, fols.
178r–80v, and Orient. 757, fols. 140r–142v;

.... also found in the
so-called "Monastic Collection", edited and translated
from mss. BritLib Orient. 764 
....

In the same Monastic
Collection the Sentences to Monks (CPG II, no.
2435) are found ...
....
In ms.
BritLib Orient. 754, fols. 182rf., the Sentences
are further followed by three letters from the
correspondence between Lucius (Lukyos) and E.,
whose second letter is divided into 21 sections;
the identification of Lucius with the addressee
of E.’s Treatise to Eulogius (CPG II, no. 2447) is,
however, doubtful (cf. Hausherr 1940)."

File: new/LIT6026NagaraBaqweet.xml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DE" when="2020-06-15">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+              <listRelation>
+                <relation name="dcterms:creator" active="LIT6026NagaraBaqweet" passive="PRS3921Evagrius"></relation>
```
Comment (eu-genia) [inline new/LIT6026NagaraBaqweet.xml:56]: I would use "saws:isAttributedToAuthor" in such cases (esp. works only attributed to certain authors in Ethiopian tradition)

File: new/LIT6027LetterofLucius.xml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DE" when="2020-06-15">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+              <listRelation>
+                <relation name="dcterms:creator" active="LIT6027LetterofLucius" passive="PRS6375Lucius"></relation>
```
Comment (eu-genia) [inline new/LIT6027LetterofLucius.xml:56]: attributedTo
cert="low"

with the desc 
the identification of Lucius with the addressee
of E.’s Treatise to Eulogius (CPG II, no. 2447) is,
however, doubtful (cf. Hausherr 1940).

Comment (eu-genia) [reply new/LIT6027LetterofLucius.xml:56]: could be helpful to add relation isAbout to link Evagrius, or maybe annotate Evagrius with a persName element, to make it appear in the ontology for E.

File: new/LIT6028LetterofEvagrius.xml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DE" when="2020-06-15">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+              <listRelation>
+                <relation name="dcterms:creator" active="LIT6028LetterofEvagrius" passive="PRS3921Evagrius"></relation>
```
Comment (eu-genia) [inline new/LIT6028LetterofEvagrius.xml:56]: attributed

Lucius to be also related to or annotated

Comment (eu-genia) [reply new/LIT6028LetterofEvagrius.xml:56]: PS bibl can also be added, e.g. here Irénée Hausherr, “Eulogios
– Loukios”, OrChrP 6, 1940, 216–220

File: new/LIT6026NagaraBaqweet.xml (hunk 1 above)
Comment (DariaElagina) [reply new/LIT6026NagaraBaqweet.xml:8]: This is the second letter, thank you for the information - I have no access to EAe at home

**Comment by DariaElagina** [comment]
updated

**Pull request merged**